src/ansys/aedt/core/visualization/advanced/rcs_visualization.py

```diff
@@ -1617,6 +1620,92 @@ def add_waterfall(
 
         self.all_scene_actors["results"]["waterfall"][waterfall_name] = rcs_mesh
 
+    @pyaedt_function_handler()
+    def add_isar_2d(
+        self,
+        plot_type="Plane",
```

Comment:
```suggestion
        plot_type="plane",
```
Since we are using lower cases afterward we should default to "plane" instead